### DISCUSSION TITLE
glossary permissions

### DISCUSSION THREAD

**Discussion by earl-warren** [WeblateOrg/weblate#11031] [General]
Hi,
Is it possible to restrict the permissions to modify a glossary on a public project? Or something similar to how strings are approved on a public project so that only glossary entries approved by reviewers are used instead of all of them. Also, it looks like anyone can delete entries in the glossary which makes it susceptible to vandalism.
Thanks for the help!

**Comment by nijel** [comment]
Yes, it is possible to adjust permissions of the default `Users` group. But this is site-wide action.
